### DISCUSSION TITLE
How to register yourself? Missing in the docs?

### DISCUSSION THREAD

**Discussion by Elekam** [firefly-iii/firefly-iii#8372] [Firefly III] [answered]
Unless Im overlooking something, I think the docs are missing how to setup your first install. Either Im not looking well enough, or the docs could be more clear/are missing it.

I setup firefly using docker, and am now seeing the login screen. I search in the docs how to create my account/login, but [this page](https://docs.firefly-iii.org/tutorials/finances/first-steps/) says "This tutorial assumes you already registered yourself, as it is explained in the [how-to guides](https://docs.firefly-iii.org/how-to/)." But when following that link I don't see any guide or direction how to first register yourself? Am I missing something?

**Comment by Elekam** [comment]
Ah I see now you just use the UI here: 
![grafik](https://github.com/firefly-iii/firefly-iii/assets/43169119/636398c9-7751-435b-b9df-086ba80f9b0e)

With the docs saying "Registered yourself" I thought it meant some special setup like for example in paperless-ngx where you have to first setup a superuser by using the CLI. I thought having to register yourself there would be so obvious that the docs meant something else. The fact still stands though that there doesn't seem to be a guide to register yourself in the how-to-guides.

**Reply by JC5** [reply]
> doesn't seem to be a guide to register yourself in the how-to-guides.

I believe people can handle this form. :)

**Reply by Elekam** [reply]
Yeah I think so too, just that the "as it is explained in the how-to-guides" is misleading, because it isn't actually explained or mentioned and makes it seem like you have to do more than you actually do?

**Reply by JC5** [reply, marked as answer]
Fair point, I've written it down!

**Reply by Elekam** [reply]
Thanks! :)

**Comment by JC5** [comment]
When you go to Firefly III for the first time, it asks you to register. If you get send to the login screen with no Register button, there is already a user in the database.
